src/components/CreateResolution/CompleteResolution.vue

```diff
@@ -0,0 +1,828 @@
+        <v-icon color="primary" style="padding-right: 5px">mdi-help-circle-outline</v-icon>
+        <span v-if="!helpToggle">{{ getCreateResolutionResource.helpSection.header }}</span>
+        <span v-else>Hide Help</span>
+      </span>
+      <section v-show="helpToggle" class="create-resolution-help">
+        <header class="create-resolution-help-header">
+          <h2>{{getCreateResolutionResource.helpSection.header}}</h2>
+        </header>
+        <span v-for="(item, index) in getCreateResolutionResource.helpSection.helpText.section1.items" :key="index">
+          <p v-if="item.type === ItemTypes.TEXT" v-html="item.value" class="mt-3"></p>
+          <v-row v-if="item.type === ItemTypes.PARTIAL_ITEMS">
+            <v-col cols="11">
+              <span v-for="(partialItem, index) in item.value" :key="index">
+                <span v-if="partialItem.type === ItemTypes.TEXT" v-html="partialItem.value"></span>
+                <a v-if="partialItem.type === ItemTypes.LINK"
+                   :href="partialItem.value.href"
+                   target="_blank"
+                >{{partialItem.value.linkText}}
+                  <v-icon dense color="primary">mdi-open-in-new</v-icon>
+                </a>
+              </span>
+            </v-col>
+          </v-row>
+          <ul v-if="item.type === ItemTypes.BULLET_LIST" class="mt-1">
+            <li v-for="(bulletItem, index) in item.value" :key="index">
+              <v-row class="mt-1">
+                <v-col v-if="item.bulletListType === BulletListTypes.CIRCLE_SMALL" cols="1" class="pt-1 pb-1">
+                  <v-icon>mdi-circle-small</v-icon>
+                </v-col>
+                <v-col v-if="item.bulletListType === BulletListTypes.ALPHABETICAL_LOWER" cols="1" class="pt-1 pb-1">
+                  ({{convertIntToChar(index)}})
+                </v-col>
+                <v-col cols="11" class="ml-n11 pt-1 pb-1">
+                  <span>{{bulletItem}}</span>
+                </v-col>
+              </v-row>
+            </li>
+          </ul>
+        </span>
+        <u class="help-btn" @click="helpToggle = !helpToggle"><small>Hide Help</small></u>
+      </section>
+    </div>
+
+    <section id="sample-resolution-section" v-if="entityFilter(CorpTypeCd.COOP)" class="mt-10">
+      <header id="sample-resolution-header">
+        <h2>{{getCreateResolutionResource.sampleFormSection.header}}</h2>
+      </header>
+      <p class="mt-2" v-html="getCreateResolutionResource.sampleFormSection.text">
+      </p>
+      <v-card flat id="sample-resolution-card" class="pt-7 pb-5 pl-6">
+        <v-row>
+          <v-col id="sample-resolution-card-left-col" cols="1" class="pt-6" >
+          </v-col>
+          <v-col id="sample-resolution-card-center-col" cols="3" class="pl-8 pt-6">
+            <img :src="previewImageSource()" class="preview-image" />
+          </v-col>
+          <v-col id="sample-resolution-card-right-col" cols="6" class="pt-6 pr-15">
```

Comment:
No action required here, as we are not building for mobile.
Just keep in mind that defining cols as such would result in all of these trying to jam into 1 line no matter what the screen size. Vue's grid system handles this pretty easily by having a prop for each size (xs, sm, lg.. etc) and cols being the default when none are defined.
Works totally fine here though, just sharing my thoughts :)